docs/03-Cloud-Native-Journey/references.adoc

```diff
@@ -1,4 +1,7 @@
 === {references}
 
```

Comment:
Das bitte auch alles in das Literaturverzeichnis, wird ab hier nicht mehr angemerkt. 🙂

Reply:
✔️